test/e2e/multikueue/e2e_test.go

```diff
@@ -363,13 +371,17 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 							Reason: "ClientConnectionFailed",
 						},
 						cmpopts.IgnoreFields(metav1.Condition{}, "LastTransitionTime", "Message"))))
-				}, util.Timeout, util.Interval).Should(gomega.Succeed())
+				}, util.LongTimeout, util.Interval).Should(gomega.Succeed())
 			})
 
 			ginkgo.By("Reconnecting worker1 container to the kind network", func() {
-				cmd := exec.Command("docker", "network", "connect", "kind", "kind-worker1-control-plane")
+				cmd := exec.Command("docker", "network", "connect", "kind", worker1Container)
 				output, err := cmd.CombinedOutput()
 				gomega.Expect(err).NotTo(gomega.HaveOccurred(), "%s: %s", err, output)
+				gomega.Eventually(func(g gomega.Gomega) {
+					g.Expect(k8sWorker1Client.List(ctx, &corev1.PodList{}, podsListOptions)).To(gomega.Succeed())
+				}, util.LongTimeout, util.Interval).Should(gomega.Succeed())
+				time.Sleep(util.LongTimeout)
```

Comment:
:), no

Reply:
For some reason, after reconnecting the container to the network, when we try to get pods, we get it with the previous values (as before disconnect). Therefore, it takes some time for the cluster to restore them and we got actually values.

Reply:
I understand but instead of  just sleeping `util.LongTimeout` we need to wait for an event that indicates the the `some time` has passed and we can continue with our test now.